stream-chat-android-client/src/main/java/io/getstream/chat/android/client/utils/PerformanceUtils.kt

```diff
@@ -3,46 +3,73 @@ package io.getstream.chat.android.client.utils
 import android.util.Log
 
 public object PerformanceUtils {
```

Comment:
NIT: We could annotate it with `InternalStreamChatApi`

Reply:
Nice catch!